pkg/cvc/v1alpha1/build.go

```diff
@@ -0,0 +1,282 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	apismaya "github.com/openebs/csi/pkg/apis/openebs.io/maya/v1alpha1"
```

Comment:
alias this to `apis`

Reply:
@AmitKumarDas apismaya was used to be in sync with the use at other places
```
apismaya "github.com/openebs/csi/pkg/apis/openebs.io/maya/v1alpha1"
apis "github.com/openebs/csi/pkg/apis/openebs.io/core/v1alpha1"
```